docs/src/token-2022.md

```diff
@@ -138,8 +141,8 @@ For information about the types and instructions, the Rust docs are available at
 
 ## Security Audits
 
-The Token-2022 Program is currently under multiple audits to ensure safety of
-funds. All audits will be published here as they are completed.
+The Token-2022 Program has been audited multiple times to ensure safety of
```

Comment:
are the lawyers cool with this language? audits don't "ensure" anything. :trollface:

Reply:
Oof yeah, that's a good point, I'll just say it's been audited multiple times